PlaybackLab/MultiplePlayersScrollingViewSampleApp/MultiplePlayersScrollingViewSampleApp.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,539 @@
+/* End PBXFileReference section */
+
+/* Begin PBXFrameworksBuildPhase section */
+		B860325E225FDE280049F604 /* Frameworks */ = {
+			isa = PBXFrameworksBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				B8AD6B4B226068F200EB71E8 /* OoyalaSkinSDK.framework in Frameworks */,
+				B86032CC225FE4EA0049F604 /* EventKit.framework in Frameworks */,
+				B86032C5225FE3580049F604 /* libxml2.tbd in Frameworks */,
+				B86032C3225FE3500049F604 /* libc++.tbd in Frameworks */,
+				B86032C1225FE3460049F604 /* libc++abi.tbd in Frameworks */,
+				B86032BF225FE2DB0049F604 /* libz.tbd in Frameworks */,
+				B86032CE225FE5040049F604 /* MediaPlayer.framework in Frameworks */,
+				B86032C8225FE4C80049F604 /* MessageUI.framework in Frameworks */,
+				B860327A225FDFAF0049F604 /* OoyalaSDK.framework in Frameworks */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXFrameworksBuildPhase section */
+
+/* Begin PBXGroup section */
+		B8603258225FDE280049F604 = {
+			isa = PBXGroup;
+			children = (
+				B8603263225FDE280049F604 /* MultiplePlayersScrollingViewSampleApp */,
+				B8603262225FDE280049F604 /* Products */,
+				B8603278225FDFAF0049F604 /* Frameworks */,
+			);
+			sourceTree = "<group>";
+		};
+		B8603262225FDE280049F604 /* Products */ = {
+			isa = PBXGroup;
+			children = (
+				B8603261225FDE280049F604 /* MultiplePlayersScrollingViewSampleApp.app */,
+			);
+			name = Products;
+			sourceTree = "<group>";
+		};
+		B8603263225FDE280049F604 /* MultiplePlayersScrollingViewSampleApp */ = {
+			isa = PBXGroup;
+			children = (
+				B8603264225FDE280049F604 /* AppDelegate.swift */,
+				B860326B225FDE2C0049F604 /* Assets.xcassets */,
+				B860326D225FDE2C0049F604 /* LaunchScreen.storyboard */,
+				B8603268225FDE280049F604 /* Main.storyboard */,
+				B8603283225FE06C0049F604 /* Models */,
+				B86032A1225FE2290049F604 /* Resources */,
+				B8603276225FDED40049F604 /* SupportFiles */,
+				B8603286225FE0B30049F604 /* Utils */,
+				B8603290225FE1070049F604 /* ViewControllers */,
+				B860328B225FE0DC0049F604 /* Views */,
+			);
+			path = MultiplePlayersScrollingViewSampleApp;
+			sourceTree = "<group>";
+		};
+		B8603276225FDED40049F604 /* SupportFiles */ = {
+			isa = PBXGroup;
+			children = (
+				B8603270225FDE2C0049F604 /* Info.plist */,
+				B8603277225FDEE00049F604 /* MultiplePlayersScrollingViewSampleApp-Bridging-Header.h */,
+			);
+			path = SupportFiles;
+			sourceTree = "<group>";
+		};
+		B8603278225FDFAF0049F604 /* Frameworks */ = {
+			isa = PBXGroup;
+			children = (
+				B86032CB225FE4EA0049F604 /* EventKit.framework */,
+				B86032C2225FE3500049F604 /* libc++.tbd */,
+				B86032C0225FE3460049F604 /* libc++abi.tbd */,
+				B86032C4225FE3580049F604 /* libxml2.tbd */,
+				B86032BE225FE2DB0049F604 /* libz.tbd */,
+				B86032CD225FE5040049F604 /* MediaPlayer.framework */,
+				B86032C6225FE3670049F604 /* MessageUI.framework */,
+				B8603279225FDFAF0049F604 /* OoyalaSDK.framework */,
```

Comment:
Look at all other sample apps, it's done in another way:
we have a separate `VendorLibraries` folder and soft links to frameworks, not to folders. It's also added in the same way to the project, not to `Frameworks`

Reply:
done